common/testutil/ssl_integration.go

```diff
@@ -15,8 +15,8 @@ func GetSSLOptions() commonOpts.SSL {
 	if testtype.HasTestType(testtype.SSLTestType) {
 		return commonOpts.SSL{
 			UseSSL:        true,
-			SSLCAFile:     "../common/db/openssl/testdata/ca.pem",
-			SSLPEMKeyFile: "../common/db/openssl/testdata/server.pem",
+			SSLCAFile:     "../common/db/openssl/testdata/ca-ia.pem",
+			SSLPEMKeyFile: "../common/db/openssl/testdata/test-client.pem",
```

Comment:
I noticed that two `.pem` files were added: `test_server.pem` and `test_client.pem`. Why are two new files needed? Why did this line change from `.../server.pem` to `.../client.pem`? Does it matter?

Reply:
The old certificate expired when I did the test. So I copied the new one from master branch. 
Yes, the change here is needed. Since the client certificate used in the test was wrong, it should not be the same as the server certificate. But this problem was not shown since we had sslAllowInvalidCertificates in server startup cmd.